components/security/authorization.rst

```diff
@@ -159,6 +159,40 @@ role::
 
     $roleHierarchyVoter = new RoleHierarchyVoter($roleHierarchy);
 
+ExpressionVoter
+~~~~~~~~~~~~~~~~~~
```

Comment:
This underline is longer than the headline

Reply:
Fixed.